client/service/src/lib.rs

```diff
@@ -120,8 +120,9 @@ where
 	let pov_recovery = PoVRecovery::new(
 		overseer_handle.clone(),
 		// We want that collators wait at maximum the relay chain slot duration before starting
-		// to recover blocks.
-		RecoveryDelay { min: core::time::Duration::ZERO, max: relay_chain_slot_duration },
+		// to recover blocks. Additionally, we wait at least half the slot time to give the
+		// relay chain the chance to increase availability.
+		RecoveryDelay { min: relay_chain_slot_duration / 2, max: relay_chain_slot_duration },
```

Comment:
Why increasing from 0 to `duration/2` helps?

Because there is only one try?
And trying later has higher chance of succes?

Reply:
This here is just a first step until we have it properly fixed: https://github.com/paritytech/cumulus/issues/2142

> Because there is only one try?

Currently yes

Reply:
Yes, you can find some additional details in the issue I linked.

Basically the chance increases that the block is available on the relay chain. The relay chain just had more time to distribute the chunks.

I plan to implement a second try too, but not in this PR.